README.md

```diff
@@ -134,6 +141,10 @@ You will also likely want to change the dates over which to run:
     DATE_FIRST_CYCL
     DATE_LAST_CYCL
 
+The deault is cold start only. You ican turn on data assimilation cycling by setting:
```

Comment:
Typos: 
 - ican -> can
 - deault -> default

This is not true for the config.sh.RRFS_dev1, so is a bit confusing. Please mention which configs exhibit this behavior.

Reply:
Updated with more details.